tests/features/capsulahub-application.feature

```diff
@@ -0,0 +1,80 @@
+
+ Feature: CapsulaHub application
+
+ Background:
+    Given a npm package called @capsulajs/capsula-hub
+    And   a default port 55555
+    And   a valid port is a number between 1024 and 65535
```

Comment:
1024 - 65535 - where did you get this range?

Reply:
@MikeSquall suggested this range and I tried to use 65536 port while testing and an error was thrown